app/routes/index.js

```diff
@@ -93,6 +93,15 @@ function createModelNotEligible (backUrl, ineligibleContent) {
   }
 }
 
+const MAYBE_ELIGIBLE = (backUrl, nextPath, maybeEligibleContent) => {
```

Comment:
Capitals should not be used in function name

Reply:
sure , it was in water project

Reply:
in water ```MAYBE_ELIGIBLE``` was a constant, not a function